tests/e2e/logs_test.go

```diff
@@ -0,0 +1,84 @@
+package e2e_test
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/rancher/elemental/tests/e2e/helpers/misc"
+)
+
+func checkRC(err error) {
+	if err != nil {
+		GinkgoWriter.Printf("%s\n", err)
+	}
+}
+
+var _ = Describe("E2E - Getting logs node", Label("logs"), func() {
+	type Binary struct {
+		Url  string
+		Name string
+	}
+
+	type getResourceLog struct {
+		Name string
+		Verb []string
+	}
+	var elementalBinary string = fmt.Sprintf("https://github.com/rancher/elemental-operator/releases/download/v%s/elemental-support_%s_linux_amd64", elementalSupportVersion, elementalSupportVersion)
+
+	It("Get the upstream cluster logs", func() {
+		By("Downloading and executing tools to generate logs", func() {
+			elemental := Binary{
+				elementalBinary,
+				"elemental-support",
+			}
+
+			logCollector := Binary{
+				"https://raw.githubusercontent.com/rancherlabs/support-tools/master/collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh",
+				"rancher2_logs_collector.sh",
+			}
+
+			_ = os.Mkdir("logs", 0755)
+			_ = os.Chdir("logs")
+			myDir, _ := os.Getwd()
+
+			var binaries []Binary = []Binary{elemental, logCollector}
+			for _, b := range binaries {
+				Eventually(func() error {
+					err := exec.Command("curl", "-L", b.Url, "-o", b.Name).Run()
```

Comment:
And in fact  as `exec.Command` only returns an `error` you can simply write `return exec.Command("curl", "-L", b.Url, "-o", b.Name).Run()`.